src/main.rs

```diff
@@ -147,7 +147,7 @@ fn run_app<B: Backend>(
 
                     Ok(())
                 })
-                .unwrap();
+                .unwrap_or(());
```

Comment:
When doing an operation like squash from the command runner, this would panic.

Reply:
unwrap removed in #75